client/.eslintrc

```diff
@@ -58,7 +58,8 @@
           "message": "@extends is to be used over @augments as it is more evocative of classes than @augments",
           "replacement": "extends"
         }
-      }
+      },
+      "mode": "typescript"
```

Comment:
This type of change should go in a separate PR in the future since it's not relevant to this PR's purpose.